app/companies/my-profile/page.tsx

```diff
@@ -1,7 +1,7 @@
 "use client";
 
 import { useRef, useState, FormEvent } from "react";
-
+import toast from "react-hot-toast";
 import DragAndDropFile from "../../components/drag-and-drop-file";
 import { Button } from "../../components/button";
```

Comment:
import { useRef, useState, FormEvent } from "react";

import toast from "react-hot-toast";

import DragAndDropFile from "../../components/drag-and-drop-file";
import { Button } from "../../components/button";